packages/blocks/src/api/factory.js

```diff
@@ -212,6 +232,57 @@ const getBlockTypesForPossibleToTransforms = ( blocks ) => {
 	return blockNames.map( ( name ) => getBlockType( name ) );
 };
 
+/**
+ * Determines whether transform is a "block" type
+ * and if so whether it is a "wildcard" transform
+ * ie: targets "any" block type
+ *
+ * @param  {Object} t the Block transform object
```

Comment:
Minor: To my prior note about alignment, it implies that we could callapse the spaces as well.

```suggestion
 * @param {Object} t the Block transform object
```